app/policies/mhv_account_creation_policy.rb

```diff
@@ -2,7 +2,7 @@
 
 MhvAccountCreationPolicy = Struct.new(:user, :mhv_account_creation) do
   def access?
-    user.loa3? && user.va_patient?
+    user.loa3?
```

Comment:
I feel like this makes it a little more confusing to read for `eligible?` in `MhvAccount` since being a VA patient is still part of the conceptual definition for being eligible.

I think the real change that needed to be made is that we don't want the user to be put into the `ineligible` state if we know for sure the reason they're unable to progress in account creation. In which case, should those transitions be written above the generic ineligible transition?

```
transitions from: ALL_STATES, to: :needs_ssn_resolution, if: :ssn_mismatch?
transitions from: ALL_STATES, to: :needs_va_patient, unless: :va_patient?
transitions from: ALL_STATES, to: :ineligible, unless: :eligible?
```

In any case, if we push these changes as they are, I think the user would skip `ineligible` to `needs_va_patient`, which is a state that's currently unhandled by the F/E.